plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -275,6 +275,15 @@ public void testCreatePartitionedTable()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testPathUriDecoding()
+    {
+        String tableName = "test_uri_table_" + randomNameSuffix();
+        registerTableFromResources(tableName, "databricks/uri", getQueryRunner());
```

Comment:
The test doesn't prove the Delta change is correct impl of Delta spec, since it can be both our writer and our reader are wrong.

@ebyhr do you know which test class would be the best for this type of a test?
we have a few `TestDeltaLakeDatabricks*Compatibility`, but none seem applicable to me

Reply:
I can't find a good existing class either. Adding a new product test class looks good to me.

Reply:
There is a bit of test coverage of this (as witnessed by the test failures in [1ca31c1](https://github.com/trinodb/trino/pull/15201/commits/1ca31c17a48e066de1b2d80ced17cb4fc3ae7c9f)). The new test added only uses the reader, and would fail with the old implementation.

I've tried adding a product test in https://github.com/trinodb/trino/pull/15201/commits/7a513880c709fcce92e70ab03d5b9b71bfd12212